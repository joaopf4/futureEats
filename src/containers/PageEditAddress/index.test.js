import React from 'react'
import renderer from 'react-test-renderer'
import {
    ProfileWrapper, ProfileHeader, ProfileText,
    ProfileTextSecond, ProfileInfos, EditIcon, ProfileInfosWrapper,
    ProfileAdressInfoWrapper, ProfileAdressInfo, HistPedidosCard, PedidoPlaceText,
    PedidoDateText, SubtotalText, HistPedidosWrapper,
    EditProfileForm, BackIcon
} from '../PageEditAddress/style'

test("Testando styled do ProfileWrapper", () => {
    const newSnapShot = renderer.create(<ProfileWrapper />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do ProfileHeader", () => {
    const newSnapShot = renderer.create(<ProfileHeader />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do ProfileText", () => {
    const newSnapShot = renderer.create(<ProfileText />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do ProfileTextSecond", () => {
    const newSnapShot = renderer.create(<ProfileTextSecond />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do ProfileInfos", () => {
    const newSnapShot = renderer.create(<ProfileInfos />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do EditIcon", () => {
    const newSnapShot = renderer.create(<EditIcon />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do ProfileInfosWrapper", () => {
    const newSnapShot = renderer.create(<ProfileInfosWrapper />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do ProfileAdressInfoWrapper", () => {
    const newSnapShot = renderer.create(<ProfileAdressInfoWrapper />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do ProfileAdressInfo", () => {
    const newSnapShot = renderer.create(<ProfileAdressInfo />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do HistPedidosCard", () => {
    const newSnapShot = renderer.create(<HistPedidosCard />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do PedidoDateText", () => {
    const newSnapShot = renderer.create(<PedidoDateText />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do SubtotalText", () => {
    const newSnapShot = renderer.create(<SubtotalText />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do HistPedidosWrapper", () => {
    const newSnapShot = renderer.create(<HistPedidosWrapper />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do EditProfileForm", () => {
    const newSnapShot = renderer.create(<EditProfileForm />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do BackIcon", () => {
    const newSnapShot = renderer.create(<BackIcon />).toJSON();

    expect(newSnapShot).toMatchSnapshot();
})


