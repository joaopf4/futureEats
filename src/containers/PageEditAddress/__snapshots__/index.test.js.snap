// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testando styled do BackIcon 1`] = `
<svg
  className="sc-fjdhpX kGFPwa"
  fill="currentColor"
  height="1em"
  stroke="currentColor"
  strokeWidth="0"
  style={
    Object {
      "color": undefined,
    }
  }
  viewBox="0 0 256 512"
  width="1em"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M31.7 239l136-136c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9L127.9 256l96.4 96.4c9.4 9.4 9.4 24.6 0 33.9L201.7 409c-9.4 9.4-24.6 9.4-33.9 0l-136-136c-9.5-9.4-9.5-24.6-.1-34z"
  />
</svg>
`;

exports[`Testando styled do EditIcon 1`] = `
<div
  className="sc-EHOje gOzjmu"
/>
`;

exports[`Testando styled do EditProfileForm 1`] = `
<form
  className="sc-jTzLTM irmnoo"
/>
`;

exports[`Testando styled do HistPedidosCard 1`] = `
<div
  className="sc-dnqmqq dEAirA"
/>
`;

exports[`Testando styled do HistPedidosWrapper 1`] = `
<div
  className="sc-VigVT fbfmFh"
/>
`;

exports[`Testando styled do PedidoDateText 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10 sc-gZMcBi lnGOOI"
/>
`;

exports[`Testando styled do ProfileAdressInfo 1`] = `
<div
  className="sc-htoDjs dTnJEb"
/>
`;

exports[`Testando styled do ProfileAdressInfoWrapper 1`] = `
<div
  className="sc-gzVnrw hZVEKp"
/>
`;

exports[`Testando styled do ProfileHeader 1`] = `
<div
  className="sc-bwzfXH dhKYig"
/>
`;

exports[`Testando styled do ProfileInfos 1`] = `
<div
  className="sc-ifAKCX elEHDe"
/>
`;

exports[`Testando styled do ProfileInfosWrapper 1`] = `
<div
  className="sc-bZQynM iavprM"
/>
`;

exports[`Testando styled do ProfileText 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10 sc-htpNat ipdBQQ"
/>
`;

exports[`Testando styled do ProfileTextSecond 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10 sc-bxivhb kYexLI"
/>
`;

exports[`Testando styled do ProfileWrapper 1`] = `
<div
  className="sc-bdVaJa jdLWYi"
/>
`;

exports[`Testando styled do SubtotalText 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10 sc-gqjmRU dCPwNO"
/>
`;
